mocked_servers/https/Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM golang:1.17 as build
+FROM golang:1.17 AS build
 WORKDIR $GOPATH/main
 COPY . .
+RUN go env -w GOPROXY=direct
+RUN GO111MODULE=on go mod download
```

Comment:
This is for mod caching?

Reply:
I think in order to achieve that you might need to copy just `go.mod` and `go.sum` as otherwise any changes in other files would cause a different layer hash a couple steps previously and force this step to execute again.  The remaining files could be copied in after this step.